orchagent/portsorch.cpp

```diff
@@ -211,6 +211,24 @@ PortsOrch::PortsOrch(DBConnector *db, vector<table_name_with_pri_t> &tableNames)
         SWSS_LOG_ERROR("Watermark flex counter groups were not set successfully: %s", e.what());
     }
 
+    string portRatePluginName = "port_rates.lua";
+
+    try
+    {
+        string portRateLuaScript = swss::loadLuaScript(portRatePluginName);
+        string portRateSha = swss::loadRedisScript(m_counter_db.get(), portRateLuaScript);
+
+        vector<FieldValueTuple> fieldValues;
+        fieldValues.emplace_back(PORT_PLUGIN_FIELD, portRateSha);
+        fieldValues.emplace_back(POLL_INTERVAL_FIELD, PORT_RATE_FLEX_STAT_COUNTER_POLL_MSECS);
+        fieldValues.emplace_back(STATS_MODE_FIELD, STATS_MODE_READ);
+        m_flexCounterGroupTable->set(PORT_RATE_COUNTER_FLEX_COUNTER_GROUP, fieldValues);
+    }
+    catch (...)
+    {
+        SWSS_LOG_WARN("Port rate flex counter group plugins was not set successfully");
```

Comment:
Can you log `e.what()` here like we've done for the watermarks? Could be helpful for debugging!

Reply:
Done